config.py

```diff
@@ -0,0 +1,1409 @@
+			if is_subset: 
+				new_data = self.runSubsetStats(subset_i, data_i, comp_i)
+
+			else:
+				# Run standard data stats
+				new_data = self.runDataStats(data_i, comp_i)
+
+			print(new_data) 
+			print("xoxoxo")   
+			print(newly_selected[index].row())
+			print("newly selected item ^")
+			print(self.nestedtree.itemFromIndex(newly_selected[index]))
+			print(new_data[0])
+			print(new_data[1])
+			print(new_data[2])
+			# self.nestedtree.itemFromIndex(newly_selected[0]).setData(0, 0, new_data[0][0])
+			for col_index in range (0, len(new_data)):
+				if (col_index > 0):
+					print("xxxxxx")
+					print(new_data[col_index])
+					self.nestedtree.itemFromIndex(newly_selected[index]).setData(col_index, 0, new_data[col_index])  
+
+			# self.nestedtree.itemFromIndex(newly_selected[index]).setData(2, 0, 100)   
+		# newly_dropped = np.setdiff1d(self.past_selected, self.selected_indices)
+		# for index in range (0, len(newly_dropped)):
+				
+		# 	# Check which view mode the tree is in to get the correct indices
+		# 	if not self.component_mode:
+		# 		data_i = newly_dropped[index].parent().row()
+		# 		comp_i = newly_dropped[index].row()
+		# 		subset_i = newly_dropped[index].parent().parent().row()
+			
+		# 	else:
+		# 		data_i = newly_dropped[index].parent().parent().row()
+		# 		comp_i = newly_dropped[index].parent().row()
+		# 		subset_i = newly_dropped[index].row() - 1
+			
+		# 	is_subset = newly_dropped[index].parent().parent().parent().row() == 1 or (self.component_mode and subset_i != -1)
+
+		# 	if is_subset:
+		# 		try:
+		# 			# Get the indices that match the component, dataset, and subset requirements
+		# 			idx_c = np.where(self.data_frame['Component'] == dc[data_i].components[comp_i].label)
+		# 			idx_d = np.where(self.data_frame['Dataset'] == dc[data_i].label)
+		# 			idx_s = np.where(self.data_frame['Subset'] == dc[data_i].subsets[subset_i].label)
+		# 			idx1 = np.intersect1d(idx_c, idx_d)
+		# 			idx2 = np.intersect1d(idx1, idx_s)
+
+		# 			self.data_frame = self.data_frame.drop(idx2)
+		# 		except:
+		# 			pass
+
+		# 	else:
+		# 		try:
+		# 		# Find the index in the table of the unchecked element, if it's in the table
+
+		# 			# Find the matching component and dataset indices and intersect them to get the unique index
+		# 			idx_c = np.where(self.data_frame['Component'] == dc[data_i].components[comp_i].label)
+		# 			idx_d = np.where(self.data_frame['Dataset'] == dc[data_i].label)
+		# 			idx_s = np.where(self.data_frame['Subset'] == '--')
+		# 			idx1 = np.intersect1d(idx_c, idx_d)
+		# 			idx2 = np.intersect1d(idx1, idx_s)
+
+		# 			# self.data_frame = self.data_frame.drop(idx2)
+		# 		except:
+		# 			pass
+		
+		# # Update the past selected indices
+		# self.past_selected = self.selected_indices
+		
+		# model = pandasModel(self.data_frame, dc)
+		
+		# self.table.setModel(model)
+	   
+		# self.table.setSortingEnabled(True)
+		# self.table.setShowGrid(False) 
+
+	def itemMasterList(self):
+
```

Comment:
In general it is good practice to try and add a docstring or a comment to all methods to make it easier to understand what's going on. Could you explain what the purpose of this method is?